src/main/java/ru/javarush/quest/impl/AnswerImpl.java

```diff
@@ -0,0 +1,62 @@
+package ru.javarush.quest.impl;
+
+import ru.javarush.quest.entity.Answer;
+import ru.javarush.quest.service.AbstractDao;
+import ru.javarush.quest.service.AnswerDao;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class AnswerImpl extends AbstractDao implements AnswerDao {
```

Comment:
Maybe it is dao?

Reply:
I replaced all names about Impl classes to Dao.